project/Build.scala

```diff
@@ -741,7 +770,7 @@ object Build {
       id = "compiler", base = file("compiler")
   ).settings(
       commonSettings,
-      publishSettings,
+      publishSettings(VersionScheme.BreakOnPatch),
```

Comment:
Why is this `BreakOnPatch`? Or maybe more specifically, what is the public API of a compiler plugin?

Reply:
A compiler plugin has *no* public API. In that sense, we can choose any version scheme. The two possible extremes are:
* `BreakOnPatch`: it signals that, even though some methods are technically public, they can break at any time.
* `"always"` (makes even 1.x and 2.x compatible): since there is officially no public API, there is no public API than can ever break, so all versions are compatible.

Of those two choices, the latter is more theoretically adequate. However, IMO the former has more practical value.